api_tests/lib_sdk/wallets/lightning.ts

```diff
@@ -0,0 +1,89 @@
+import { pollUntilMinted, Wallet } from "./index";
+import { Asset } from "../asset";
+import BigNumber from "bignumber.js";
+import { Lnd } from "../lnd";
+import { HarnessGlobal } from "../../lib/util";
+import { Logger } from "log4js";
+import { E2ETestActorConfig } from "../../lib/config";
+import { BitcoinWallet } from "./bitcoin";
+
+declare var global: HarnessGlobal;
+
+export class LightningWallet implements Wallet {
+    public static async newInstance(
+        bitcoinWallet: BitcoinWallet,
+        logger: Logger,
+        logDir: string,
+        actorConfig: E2ETestActorConfig
+    ) {
+        const bitcoindDataDir = global.bitcoind.getDataDir();
```

Comment:
:mega: Don't access globals in here please.

Reply:
I am doing a separate PR to remove `bitcoind` from `GlobalHarness`.